### PR TITLE
Updates for v0.27.15

### PR DESCRIPTION
**Pull request by pseudoyim**
Felix (the [maintainer](https://github.com/fzumstein) of [xlwings](https://github.com/xlwings/xlwings)) asked us to build this specific version ([see Jira](https://anaconda.atlassian.net/browse/PKG-369)).
Updated the specs according to what was in [the upstream conda-forge recipe](https://github.com/conda-forge/xlwings-feedstock/blob/02407e7964ac6bc8094fac3f3f2ab2d50e159291/recipe/meta.yaml) (also maintained by Felix).

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,21 +13,21 @@ source:
...
     - python
+    - pip
     - setuptools
     - wheel
   run:
     - python
-    - pywin32 >=224  # [win and py>=37]
-    - pywin32  # [win and py36]
-    - psutil >=2.0.0  # [osx]
+    - pywin32  # [win]
```
Comment (marcoesters) [inline recipe/meta.yaml:29]: `pywin` still has version restrictions: https://github.com/xlwings/xlwings/blob/main/setup.py#L23

**Review by marcoesters — COMMENTED** [review]
Sooner or later we will need the newest `maturin` in `defaults`: https://github.com/xlwings/xlwings/blob/main/pyproject.toml#L2

File: recipe/meta.yaml (hunk 1 above)
Comment (pseudoyim) [reply recipe/meta.yaml:29]: Updated. Thanks!

**Review by marcoesters — APPROVED** [review]
LGTM

**Pull request merged**